### PR TITLE
Box 118 create dev device

### PR DESCRIPTION
**Pull request by danielkboyer**
This can be a confusing PR, if you have questions message me.

### CODE REVIEW

File: website/boxheadsweb/manager/devicemanager.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    homeScore: 108,
+    awayScore: 105,
+    qtr: 4,
+    homeTeamName: "Dallas Maviricks",
+    homeTeamAbrv: "DAL",
+    awayTeamName: "Denver Nuggets",
+    awayTeamAbrv: "DEN",
+    minutes: 2,
+    seconds: 3,
+  },
```
Comment (ktwitchn) [inline website/boxheadsweb/manager/devicemanager.ts:23]: Did you hard code this just to test it out for now?

File: website/boxheadsweb/pages/device/livedevice.tsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                className={labelStyles.title}
+                title={`${formatTime(data.gEvent.minutes)}:${formatTime(
+                  data.gEvent.seconds
+                )}`}
+              ></BoxheadsLabel>
+            </div>
+            <div className={styles.scoreBox}>
+              <div className={styles.doubleText}>
+                <BoxheadsLabel
+                  className={[labelStyles.title, styles.scoreText].join(" ")}
```
Comment (ktwitchn) [inline website/boxheadsweb/pages/device/livedevice.tsx:94]: I didn't see .scoreText in your styles sheet. I think the design Tom just put out has the scores the same size as the key stats below. Maybe just use the same style as those...

**Review by ktwitchn — APPROVED** [review]
Sorry it took so long getting around to this review. Mostly I just commented on the css for the live stats box (compared to the model that Tom just barely put up).

File: website/boxheadsweb/pages/device/livedevice.tsx (hunk 2 above)
Comment (danielkboyer) [reply website/boxheadsweb/pages/device/livedevice.tsx:94]: nice catch, I removed those unused styles.

**Comment by danielkboyer** [comment]
I think i'm going to close this PR without merging and wait for thomas to release the mocks.

**Pull request closed without merging**